ThirdPartyNoticeText.txt

```diff
@@ -85,12 +85,12 @@ DOM
 W3C License
 This work is being provided by the copyright holders under the following license.
 By obtaining and/or copying this work, you (the licensee) agree that you have read, understood, and will comply with the following terms and conditions.
-Permission to copy, modify, and distribute this work, with or without modification,�for any purpose and without fee or royalty is hereby granted, provided that you include the following 
+Permission to copy, modify, and distribute this work, with or without modification, for any purpose and without fee or royalty is hereby granted, provided that you include the following 

```

Comment:
<img width="151" alt="image" src="https://user-images.githubusercontent.com/4097841/224143965-41d7622e-a54e-43b3-ae9d-d8a387923e3a.png">

No idea *what's* going on here.

Reply:
Meanwhile in Outlook 😃 

![image](https://user-images.githubusercontent.com/5341706/224144320-83d3340a-693a-4686-b9be-555664aa8dbe.png)

Reply:
I'm starting to think this file is actually cursed.

Reply:
There's actually an `0xC2` byte at this position
![image](https://user-images.githubusercontent.com/6685088/224152939-57ead8a8-f657-4450-b3a9-30520092e21a.png)

Reply:
And here I thought I had cracked the code.